### PR TITLE
qt: Add option to choose not to start the wallet minimised.

### LINKED ISSUE #1782: Option to start the wallet unminimised?
Maybe there could be an option to not hide the wallet on system startup.

### PR DESCRIPTION
**Pull request by nathanielcwm**

### CODE REVIEW

File: src/qt/forms/optionsdialog.ui [hunk 1]
```diff
@@ -86,6 +86,16 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="gridcoinAtStartupminimised">
```
Comment (barton2526) [inline src/qt/forms/optionsdialog.ui]: this is not proper camelCase

Comment (nathanielcwm) [reply src/qt/forms/optionsdialog.ui]: would gridcoinAtStartupMinimised be proper?

**Review by jamescowens — CHANGES_REQUESTED** [review]
The state of the "Start Gridcoin silently on system login" is irrelevant if the "Start Gridcoin on system login" is unchecked. When "Start Gridcoin on system login" is unchecked, "Start Gridcoin silently on system login" should be greyed out. I don't mind you retaining the state of that checkbox independently (i.e. allowing the "Start Gridcoin silently on system login" check to remain) even if the "Start Gridcoin silently on system login" is unchecked, but it should be greyed out. Quite frankly you should probably shorten it to "Minimized" and indent it so that it is clearly viewed as a subordinate setting.

**Comment by nathanielcwm** [comment]
The last commit should have greyed out the box if the start on system startup box isn't enabled but i'll check after I wake up l8r.

**Comment by nathanielcwm** [comment]
@jamescowens ah i forgot to disable the box by default

**Comment by jamescowens** [comment]
Ok great. I will try to get around to testing this again tomorrow. Nice refinement.

**Comment by jamescowens** [comment]
Turns out to solve this was a little trickier than I thought. I took your PR, merged it to dev (at the head) and then implemented some changes and created a new PR. Closing this one. I will probably squash the commits to a little more logical form.

**Pull request closed without merging**